src/runtime/threading_backend.cc

```diff
@@ -159,14 +159,8 @@ class ThreadGroup::Impl {
         CPU_SET(sorted_order_[sorted_order_.size() - i - 1], &cpuset);
       }
     } else {
-      int big_count = big_count_;
-      // Imagine our x86 has cores 0 - 7
-      // physical cores are 0 - 3, logical cores are 4 - 7, big_count_ is 8
-      // we wish we run on physical cores, not logical cores to avoid contention issue.
```

Comment:
The big count might be necessary for the big.little ARM targets

Reply:
Is this possible to detect big.LITTLE somewhere?

Reply:
OK, after taking a closer look at the code it may not be the problem